ReleaseTooling/Template/README.md

```diff
@@ -24,37 +24,32 @@ Xcode 15.2 or newer is required.
 To integrate a Firebase SDK with your app:
 
 1. Find the desired SDK from the list within `METADATA.md`.
-2. Make sure you have an Xcode project open in Xcode.
-3. In Xcode, hit `⌘-1` to open the Project Navigator pane. It will open on
+1. Make sure you have an Xcode project open in Xcode.
```

Comment:
Since this file gets copied into the zip, and might be opened in a text editor or Xcode (i.e., devs might be seeing this a raw Markdown), I wonder if we should leave these as 1,2,3,4,...?

Reply:
+1

Reply:
good catch!